### PR TITLE
Winner Modal without replay or integration

### PR DESCRIPTION
**Pull request by emilyseibert**
Most feature work for issue #30 
* update overlay to be over arena for winner modal
* hard coded values / integration for testing

Does not include integration with end of game state

### CODE REVIEW

File: less/components/modals/winner-modal.less [hunk 1]
```diff
@@ -0,0 +1,41 @@
+@import '../../colors.less';
+@import '../../objects/button.less';
+
+.modal-container {
+    .modal-overlay {
+         width: 600px; // dependent on width of arena
```
Comment (emilyseibert) [inline less/components/modals/winner-modal.less]: this gets applied to every modal so this logic will have to move

File: src/cljs/wombats_web_client/panels/game-play.cljs [hunk 2]
```diff
@@ -94,7 +95,7 @@
   (let [dimensions (get-arena-dimensions)]
     ;; TODO This should come from the router
     (reset! game-id (get-game-id))
-
+    ; (re-frame/dispatch [:set-modal #(winner-modal "green" "Wilma" "emilyseibert")])
```
Comment (oconn) [inline src/cljs/wombats_web_client/panels/game-play.cljs]: of course you always win 😜

**Comment by emilyseibert** [comment]
@oconn @dehli made changes

**Comment by dehli** [comment]
What's the reason for having both a special modal and a regular modal?

**Comment by oconn** [comment]
Would it be possible to pass an additional argument to hide the underlay?

**Pull request merged**